packages/mobile/src/components/now-playing-drawer/NowPlayingDrawer.tsx

```diff
@@ -92,18 +95,31 @@ const NowPlayingDrawer = ({
 
   useEffect(() => {
     if (isOpen) {
+      setIsSwipedClosed(false)
       StatusBar.setHidden(true, 'fade')
     } else {
       StatusBar.setHidden(false, 'fade')
     }
   }, [isOpen])
 
-  const onDrawerClose = useCallback(() => {
+  const handleDrawerClose = useCallback(() => {
     springToValue(playBarOpacityAnim, 1, DrawerAnimationStyle.SPRINGY)
-    onClose()
     setIsPlayBarShowing(true)
     onCloseProp()
-  }, [setIsPlayBarShowing, onClose, onCloseProp, playBarOpacityAnim])
+  }, [setIsPlayBarShowing, onCloseProp, playBarOpacityAnim])
+
+  const handleDrawerCloseFromSwipe = useCallback(() => {
```

Comment:
Nice! We haven't implemented clicking on the track artwork/title/artist name yet, so we'll re-use this fn there too

Reply:
Or rather, the other close version